src/foam/u2/crunch/CrunchController.js

```diff
@@ -36,6 +38,25 @@ foam.CLASS({
     { name: 'CANNOT_OPEN_PENDING', message: 'This capability is awaiting approval, updates are not permitted at this time.' }
   ],
 
+  properties: [
+    {
+      name: 'activeIntercepts',
+      class: 'Array',
+      documentation: `
+        Since permissions may be checked during asynchronous calls,
+        it is possible that the same intercept view will be requested
+        twice in a short period of time. Keeping a a map of active
```

Comment:
```suggestion
        twice in a short period of time. Keeping a map of an active
```

Reply:
would recommend not commiting - and just reading over this text to fix - above is only one suggestion but 
> intercept **view** is done to prevent two intercept views being
> open**ed** for the same permission (as this would be confusing for

NOT BLOCKING

Reply:
Fixed the typo of typing `a` twice,

but regarding the rest of the text I wrote it in plural form intentionally. If two unrelated intercepts occur than both should open, and in this case both would be considered "active"